packages/steiger/src/shared/globs/merge-glob-groups.ts

```diff
@@ -0,0 +1,17 @@
+import { GlobGroup } from './types'
+
+export function mergeGlobGroups(globs: Array<GlobGroup>) {
+  return globs.reduce((acc, { files, ignores }) => {
+    const globGroup: GlobGroup = {}
+
+    if (files) {
+      globGroup.files = [...(acc.files || []), ...files]
+    }
+
+    if (ignores) {
+      globGroup.ignores = [...(acc.ignores || []), ...ignores]
+    }
+
+    return globGroup
+  })
+}
```

Comment:
issue: this function is throwing away globs if `files` or `ignores` is not present in the last element:
```ts
mergeGlobGroups({ files: ['*'], ignores: ['*'] }, {}) // returns {}
```

Reply:
Oh, yeah, I forgot to spread the previous result. Thanks for catching that!